CONTRIBUTING.md

```diff
@@ -3,18 +3,16 @@
 Building Cabal for hacking
 --------------------------
 
-The current recommended way of developing Cabal is to use the
-`v2-build` feature which [shipped in cabal-install-1.24](http://blog.ezyang.com/2016/05/announcing-cabal-new-build-nix-style-local-builds/).
 If you use the latest version of cabal published on Hackage, it is sufficient to run:
 
 ```
-cabal v2-build cabal
+cabal build cabal
```

Comment:
Should that be `Cabal`?

Reply:
I doubt it. If so, I'd say that's a separate problem from using the `v2-` version of commands.

Reply:
It's good. Believe it or not, `cabal build cabal` does the right thing (builds the exe).